src/test/java/teammates/sqlui/webapi/DeleteFeedbackSessionActionTest.java

```diff
@@ -0,0 +1,78 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.time.Duration;
+import java.time.Instant;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.FeedbackSession;
+import teammates.ui.output.MessageOutput;
+import teammates.ui.webapi.DeleteFeedbackSessionAction;
+import teammates.ui.webapi.JsonResult;
+
+/**
+ * SUT: {@link DeleteFeedbackSessionAction}.
+ */
+public class DeleteFeedbackSessionActionTest extends BaseActionTest<DeleteFeedbackSessionAction> {
+
+    private FeedbackSession feedbackSession;
+    private Course course;
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.SESSION;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return DELETE;
+    }
+
+    @BeforeMethod
+    void setUp() {
+        course = generateCourse1();
+        feedbackSession = generateSession1InCourse(course);
+    }
+
+    @Test
+    public void testDeleteFeedbackSessionAction_deleteFeedbackSessionAction_shouldPass() throws Exception {
```

Comment:
we may want to add the case where nothing is deleted

Reply:
ok, will do so